dev-libs/libliftoff/libliftoff-0.5.0.ebuild

```diff
@@ -0,0 +1,20 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit meson
+
+DESCRIPTION="Lightweight KMS plane library"
+HOMEPAGE="https://gitlab.freedesktop.org/emersion/libliftoff"
+SRC_URI="https://gitlab.freedesktop.org/emersion/${PN}/-/releases/v${PV}/downloads/${P}.tar.gz"
+LICENSE="MIT"
+SLOT="0/$(ver_cut 1-2)"
```

Comment:
Breaks the API, given it's likely to be recurrent I added this. Another option is to manually check if there are breaking changes.

Reply:
Thanks. I left this line as it was though. The API has broken, but we don't know that the ABI has, and that's what counts here. If the soname hasn't changed (it's still libliftoff.so.0) then we will assume the ABI has not broken. I briefly ran gamescope against 0.5.0 without rebuilding and found that it still worked. If we find breakage later, we can address it then.